authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc

```diff
@@ -0,0 +1,734 @@
+$ aws iam get-role-policy --role-name openshift-machine-api-aws-cloud-credentials --policy-name openshift-machine-api-aws-cloud-credentials
+----
++
+[source,json]
+----
+{
+    "RoleName": "openshift-machine-api-aws-cloud-credentials",
+    "PolicyName": "openshift-machine-api-aws-cloud-credentials",
+    "PolicyDocument": {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "ec2:CreateTags",
+                    "ec2:DescribeAvailabilityZones",
+                    "ec2:DescribeDhcpOptions",
+                    "ec2:DescribeImages",
+                    "ec2:DescribeInstances",
+                    "ec2:DescribeSecurityGroups",
+                           "ec2:DescribeSubnets",
+        "ec2:DescribeVpcs",
+                    "ec2:RunInstances",
+                    "ec2:TerminateInstances",
+                    "elasticloadbalancing:DescribeLoadBalancers",
+                    "elasticloadbalancing:DescribeTargetGroups",
+                    "elasticloadbalancing:RegisterInstancesWithLoadBalancer",
+                    "elasticloadbalancing:RegisterTargets",
+                    "iam:PassRole",
+                    "iam:CreateServiceLinkedRole"
+                ],
+                "Resource": "*"
+            },
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "kms:Decrypt",
+                    "kms:Encrypt",
+                    "kms:GenerateDataKey",
+                    "kms:GenerateDataKeyWithoutPlainText",
+                    "kms:DescribeKey"
+                ],
+                "Resource": "*"
+            },
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "kms:RevokeGrant",
+                    "kms:CreateGrant",
+                    "kms:ListGrants"
+                ],
+                "Resource": "*",
+                "Condition": {
+                    "Bool": {
+                        "kms:GrantIsForAWSResource": true
+                    }
+                }
+            }
+        ]
+    }
+}
+----
+. Prepare to run the {product-title} installer:
+
+.. Create the `install-config.yaml` file:
++
+[source,terminal]
+----
+$ ./openshift-install create install-config
+----
+
+.. Configure the cluster to install with the CCO in manual mode:
++
+[source,terminal]
+----
+$ echo "credentialsMode: Manual" >> install-config.yaml
+----
+
+.. Create install manifests:
++
+[source,terminal]
+----
+$ ./openshift-install create manifests
+----
+
+.. Create a `tls` directory, and copy the private key generated previously there:
++
+[NOTE]
+====
+The target file name is important.
+====
++
+[source,terminal]
+----
+$ mkdir tls ; cp <path-to-service-account-signer> ./tls/bound-service-account-signing-key.key
+----
+
+.. Create a custom `Authentication.config.openshift.io` CR with the file name `cluster-authentication-02-config.yaml`:
```

Comment:
`Authentication.config.openshift.io` CR
Is this a convention to include the API in a CR title?